zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/schema/descriptors/ComponentTemplateDescriptor.java

```diff
@@ -7,18 +7,6 @@
  */
 package io.camunda.exporter.schema.descriptors;
 
-public interface ComponentTemplateDescriptor {
+public interface ComponentTemplateDescriptor extends TemplateDetails {
```

Comment:
is this currently used anywhere?

Reply:
Actually no, I added it because I thought component templates would be created, but if they aren't (in that settings don't require them) then this and `createComponentTemplate` in the `SearchEngineClient` can also be removed so its pending on the other point regarding component templates

Reply:
you can remove it now. we shall recreate it in the future if we revisit the decision about component templates

Reply:
removed